src/datasets/non_desktop/apple_app_store/reference.md

```diff
@@ -0,0 +1,115 @@
+
+# Data Reference
+
+## Schema
+```
+metrics_by_[dimension] tables
+root
+    |- date: date
+    |- app_name: string
+    |- [app_referrer | app_version | campaign | platform | platform_version | region | source | storefront | web_referrer]: string
+    |- app_units: integer
+    |- impressions: integer
+    |- impressions_unique_device: integer
+    |- product_page_views: integer
+    |- product_page_views_unique_device: integer
+    |- active_devices_opt_in: integer
+    |- active_devices_last_30_days_opt_in: integer
+    |- deletions_opt_in: integer
+    |- installations_opt_in: integer
+    |- sessions_opt_in: integer
+
+```
+## Example Queries
+
+##### Calculate App Store Activity for a given day by app
+
+```sql
+SELECT
+  date,
+  app_name,
+  SUM(impressions_unique_device) as unique_device_impressions,
+  SUM(product_page_views_unique_device) as unique_device_page_views,
+  SUM(app_units) as installs,
+  SAFE_DIVIDE(SUM(product_page_views_unique_device), SUM(impressions_unique_device)) as unique_device_page_view_rate,
+  SAFE_DIVIDE(SUM(app_units), SUM(product_page_views_unique_device)) as install_rate
+FROM
+  `moz-fx-data-marketing-prod.apple_app_store.metrics_total`
+WHERE
+  date = "2020-08-20"
+GROUP BY
+  date, app_name
+
+  ```
+
+##### Calculate App Store Activity for a given day and app by source
+
+```sql
+SELECT
+  date,
+  app_name,
+  source,
+  SUM(impressions_unique_device) as unique_device_impressions,
+  SUM(product_page_views_unique_device) as unique_device_page_views,
+  SUM(app_units) as installs,
+  SAFE_DIVIDE(SUM(product_page_views_unique_device), SUM(impressions_unique_device)) as unique_device_page_view_rate,
+  SAFE_DIVIDE(SUM(app_units), SUM(product_page_views_unique_device)) as install_rate
+FROM
+  `moz-fx-data-marketing-prod.apple_app_store.metrics_by_source`
+WHERE
+  date = "2020-08-20"
+  AND app_name = "Firefox"
+GROUP BY
+  date, app_name, source
+```
+
+## Scheduling
+The job to retrieve the raw data from the Apple App Store can be found in [github](https://github.com/mozilla/app-store-analytics-export). The export results in individual metrics grouped by a single dimension. These exports are initially loaded into the [`apple_app_store_exported`](https://console.cloud.google.com/bigquery?project=moz-fx-data-marketing-prod&p=moz-fx-data-marketing-prod&d=apple_app_store_exported&page=dataset) datasource. The exports are scheduled in [`airflow`](https://github.com/mozilla/telemetry-airflow/blob/master/dags/app_store_analytics.py). The job to create the derived tables found in [`moz-fx-data-marketing-prod.apple_app_store`](https://console.cloud.google.com/bigquery?project=moz-fx-data-marketing-prod&p=moz-fx-data-marketing-prod&d=apple_app_store&page=dataset) can be found in [`bigquery-etl`](https://github.com/mozilla/bigquery-etl/tree/master/sql/apple_app_store).
```

Comment:
For the second sentence I think this would be clearer:
```
The exported results are individual metrics grouped by a single dimension.
```

The github suggestion is a little long.

Reply:
Changed